manifests/base/application-controller-deployment/argocd-application-controller-deployment.yaml

```diff
@@ -264,3 +264,5 @@ spec:
           items:
           - key: controller.profile.enabled
             path: profiler.enabled
+      nodeSelector:
```

Comment:
What is the concern about adding this globally? 

I'm not seeing a downside to applying this as it will have no effect on clusters that only have Linux nodes.

Reply:
I'm a bit ignorant about multi-OS clusters. Do we know with reasonable certainty that `kubernetes.io/os` will be set for all nodes for all relevant versions of Kubernetes across all major vendors? Or is it just a convention that may or may not be in place?

Reply:
In all current kubernetes clusters (version 1.14 (2019) to present (v1.31)), the `kubernetes.io/os` label is automatically set. Previous to that, the label was `beta.kubernetes.io/os`. Versions since 1.14 seem to have both labels for reverse compatibility, but officially, beta is deprecated.